ts/features/itwallet/issuance/store/actions/index.ts

```diff
@@ -8,6 +8,11 @@ export const itwRemoveIntegrityKeyTag = createStandardAction(
   "ITW_REMOVE_INTEGRITY_KEY_TAG"
 )<void>();
 
+export const itwIntegrityServiceReady = createStandardAction(
+  "ITW_INTEGRITY_SERVICE_READY"
+)<boolean>();
```

Comment:
Since this action has a boolean value parameter, its name should follow the standard setter naming convention. What do you think?

```suggestion
export const itwIntegritySetServiceIsReady = createStandardAction(
  "ITW_INTEGRITY_SET_SERVICE_IS_READY"
)<boolean>();
```

Reply:
Sure, thanks!